php/libraries/Candidate.class.inc

```diff
@@ -681,6 +681,33 @@ class Candidate
         return $vLabel;
     }
 
+    /**
+     * Returns list of consents and value of each for this candidate
+     *
+     * @param int $candID Candidate for which consent details are required
+     *
+     * @return array List of consents and their associated values for this candidate.
+     *               The keys of the arrays are the IDs of the consents
+     */
+    static function getConsent($candID)
```

Comment:
Having a static method on the candidate class which takes the candidate id as its only parameter is silly.

This should be a non-static method.